runtime/vm/jvminit.c

```diff
@@ -3893,6 +3893,15 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		}
 	}
 
+	{
+		IDATA enableJVMRestorePortableeMode = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXENABLEJVMRESTOREPORTABLEMODE, NULL);
+		if (enableJVMRestorePortableeMode > 0) {
```

Comment:
Ya I was thinking about whether there is a use case for `VMOPT_XXDISABLEVMRESTOREPORTABLEMODE`, because we wouldn't respect it if `VMOPT_XXENABLECRIUNONPORTABLEMODE` was specified. 

Perhaps its just a matter of documenting it clearly.

Reply:
Ill add the standard checks here.